scripts/kvm-test.py

```diff
@@ -571,7 +572,7 @@ def install(ctx):
     if os.path.exists(ctx.target):
```

Comment:
Maybe we want ctx.vmstate here as well. If we pass `--no-disk` (e.g., nvme/tcp), this condition will always be false.

Reply:
Fixed